### PR TITLE
feat: show notifications

### PR DESCRIPTION
**Pull request by GusBaamonde**
show-bell

Solved the :after data-count="0" @cbernat

### CODE REVIEW

File: assets/scss/templates/_header.scss [hunk 1]
```diff
@@ -251,10 +251,15 @@
...
   transition: all 0.3s ease;
 
-&.helper--right{
+&.helper--right {
   right: -12px;
 }
 
+&.notifications {
+    padding: 25px;
+    font-size: 13px;
```
Comment (cbernat) [inline assets/scss/templates/_header.scss]: I think we have a variable for this font-size right?

File: assets/templates/header.html [hunk 2]
```diff
@@ -69,10 +69,11 @@
           <ul class="nav-right-main--list">
             <li>
             <a href="#" class="user-menu--open active" data-count="1"><span class="ms-icon icon-notification"></span></a>
-            <div class="user-menu helper--right">
+            <div class="user-menu helper--right notifications">
               <h5>NUEVO: Comportamiento en Sitio</h5>
```
Comment (andresmoschini) [inline assets/templates/header.html]: It is missing changing this structure, right?

File: assets/scss/templates/_header.scss [hunk 3]
```diff
@@ -256,8 +256,8 @@
 }
 
 &.notifications {
-    padding: 25px;
-    font-size: 13px;
+    padding: $dp-spaces--lv4;
+    font-size: $dp-sizing--lvl2;
```
Comment (cbernat) [inline assets/scss/templates/_header.scss]: ❤️

**Comment by cbernat** [comment]
I do not see free user notificiation here. The one with upgrade link.

File: assets/templates/header.html (hunk 2 above)
Comment (GusBaamonde) [reply assets/templates/header.html]: Yes, just change it with the message structure that brings backend

**Comment by GusBaamonde** [comment]
> I do not see free user notificiation here. The one with upgrade link.

El contenido del mensaje no viene por backend?

**Comment by cbernat** [comment]
> > I do not see free user notificiation here. The one with upgrade link.
> 
> El contenido del mensaje no viene por backend?

Yes it does, but I thought we were implementing free message first, that's all, should be similar.

**Pull request merged**